world-server/src/world/player/PlayerHandlers.ts

```diff
@@ -82,6 +85,13 @@ export class PlayerHandler implements Messages.MsgHandler {
                 this.players.splice(index, 1);
                 Router.emit(new BroadcastMessage(message));
                 break;
+            
+            case 'player.shoot':
```

Comment:
I'd suggest moving bullets to a separate handler for readability and manageability.